shuup/core/suppliers/base.py

```diff
@@ -26,10 +26,15 @@
 @lru_cache()
 def get_supported_product_kinds_for_module(module_identifier: str) -> Iterable[ProductKindSpec]:
     specs = []
+    default_specs = []
```

Comment:
this also doesn't make any sense

Reply:
Why not?
First we will return the product specs with and exact match. Whit this I mean a supplier module where the identifier is specified in the `supported_supplier_modules` array in the product spec.
Then if we don't find any exact matches we will return all the product specs that don't have any `supported_supplier_modules`